packages/trie/src/checkpointDb.ts

```diff
@@ -0,0 +1,162 @@
+import { LevelUp } from 'levelup'
+import { DB } from './db'
+const level = require('level-mem')
+
+export const ENCODING_OPTS = { keyEncoding: 'binary', valueEncoding: 'binary' }
+
+export type Checkpoint = {
+  keyValueMap: Map<Buffer, Buffer | null>,
+  root: Buffer
+}
+
+export type BatchDBOp = PutBatch | DelBatch
+export interface PutBatch {
+  type: 'put'
+  key: Buffer
+  value: Buffer
+}
+export interface DelBatch {
+  type: 'del'
+  key: Buffer
+}
```

Comment:
these can be imported from `db.ts`

Reply:
Ah yep thanks, I actually copied the original and then removed stuff. Should indeed import from the original 👍